src/styles/block/calendar.scss

```
@@ -0,0 +1,87 @@
+.calendar {
+  width: ($size--block * 7) + 10;
```

```suggestion
  width: $size--block * 7;
```
for what additional 10px?